warehouse/models/mart/payments/fct_elavon__transactions.sql

```diff
@@ -0,0 +1,98 @@
+{{ config(materialized='table') }}
+
+WITH stg_elavon__transactions AS (
+    SELECT * FROM {{ ref('stg_elavon__transactions') }}
+),
+
+remove_special_characters AS (
+    SELECT
+
+        * EXCEPT (fund_amt, batch_amt, amount, surchg_amount, convnce_amt, payment_date, transaction_date, settlement_date),
+
+        CAST(REGEXP_REPLACE(fund_amt, r'\$|,', '') as NUMERIC) AS fund_amt,
```

Comment:
For these since it is repeated, I think it might be worth making a macro -- could be called `elavon_dollar_amount_regex` or something, just so if someone needs to tweak they can do so in one place